main_test.py

```diff
@@ -0,0 +1,42 @@
+from statistics import mean
```

Comment:
Is this script a test script from you? If so I think you can remove it, at least in the main branch

Reply:
Yes this was just for testing, removed.